### PR TITLE
MINOR: Kafka Streams updates for 2.7.0 release

### PR DESCRIPTION
**Pull request by bbejeck**
Added `upgrade-system-tests-27` module, updated `settings.gradle` and `build.gradle` for the new module, added the latest release to the `streams_upgrade_test.py`

For testing, I ran `streams:testAll` locally and everything passed.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: tests/kafkatest/tests/streams/streams_upgrade_test.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 # broker 0.10.0 is not compatible with newer Kafka Streams versions
 broker_upgrade_versions = [str(LATEST_0_10_1), str(LATEST_0_10_2), str(LATEST_0_11_0), str(LATEST_1_0), str(LATEST_1_1), \
                            str(LATEST_2_0), str(LATEST_2_1), str(LATEST_2_2), str(LATEST_2_3), \
-                           str(LATEST_2_4), str(LATEST_2_5), str(LATEST_2_6), str(DEV_BRANCH)]
+                           str(LATEST_2_4), str(LATEST_2_5), str(LATEST_2_6), str(LATEST_27), str(DEV_BRANCH)]
```
Comment (mjsax) [inline tests/kafkatest/tests/streams/streams_upgrade_test.py]: `LATEST_27` musts be imported above.

**Review by mjsax — COMMENTED** [review]
LGTM, but we should trigger a system test run before merging to make sure we did not miss anything.

**Comment by bbejeck** [comment]
kicked off system test ~https://jenkins.confluent.io/job/system-test-kafka-branch-builder/4326/~

**Comment by bbejeck** [comment]
Previous system test failed due to bad variable ref.

New system test https://jenkins.confluent.io/job/system-test-kafka-branch-builder/4328/

**Pull request merged**